lib/v2/domain-shared/app_constants.dart

```diff
@@ -9,7 +9,7 @@ const String ios_bundle_id = 'com.joinseeds.seedslight';
 const String ios_app_store_id = '1507143650';
 
 /// Accounts
-const String account_seeds = 'accts.seeds';
+const String account_accounts = 'accts.seeds';
```

Comment:
maybe we should rename these...

contract_.. . for contracts 

alliance is not a contract... IDK what that is...